src/codegeneration/FromOptionsTransformer.js

```diff
@@ -57,18 +56,20 @@ import {options, transformOptions} from '../Options.js';
 export class FromOptionsTransformer extends MultiTransformer {
   /**
    * @param {ErrorReporter} reporter
-   * @param {UniqueIdentifierGenerator=} idGenerator
+   * @param {Options} options
    */
-  constructor(reporter, idGenerator = new UniqueIdentifierGenerator()) {
+  constructor(reporter, options) {
     super(reporter, options.validate);
+    var idGenerator = new UniqueIdentifierGenerator();
 
     var append = (transformer) => {
       this.append((tree) => {
-        return new transformer(idGenerator, reporter).transformAny(tree);
+        return new transformer(idGenerator, reporter, options).
+            transformAny(tree);
       });
     };
 
-    if (transformOptions.blockBinding) {
+    if (options.transformView('blockBinding')) {
```

Comment:
Yes. Options should have a getter for a transformOptions that returns this "proxy" object. 

See https://github.com/arv/traceur-compiler/compare/options-pass-to-transformers